install/installer/pkg/components/ide-service/ide_config_configmap.go

```diff
@@ -49,7 +49,7 @@ func ideConfigConfigmap(ctx *common.RenderContext) ([]runtime.Object, error) {
 	}
 
 	jbPluginImage := ctx.ImageName(ctx.Config.Repository, ide.JetBrainsBackendPluginImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.JetBrainsBackendPluginImage.Version)
-	jbPluginLatestImage := ctx.ImageName(ctx.Config.Repository, ide.JetBrainsBackendPluginImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.JetBrainsBackendPluginLatestImage.Version)
+	jbPluginLatestImage := ctx.ImageName(ctx.Config.Repository, ide.JetBrainsBackendPluginImage, "latest")
```

Comment:
I think this change is making the build fail.
Not sure if this change is needed because the plugin was up to date on Preview Environments even when the IDEs we not using the latest image.

Reply:
Build succeeded, so it was not the problem. I'm marking this as resolved.